tensorlayer/layers/binary.py

```diff
@@ -24,6 +32,71 @@ def quantize(x):
         return tf.sign(x)
 
 
+def quantize_dorefa(x, k):
+    G = tf.get_default_graph()
+    n = float(2**k - 1)
+    with G.gradient_override_map({"Round": "Identity"}):
+        return tf.round(x * n) / n
+
+def fw(x, force_quantization=False):
```

Comment:
can we use a more meaningful names? If this function is used only internally, we better to start the name with a underscore.